test/system/test_tpm2_create_all.sh

```diff
@@ -61,3 +61,17 @@ for pCtx in `ls ctx.cpri*`
     done
 done
 
+return_val=$?
+echo "f28230c080bbe417141199e36d18978228d8948fc10a6a24921b9eba6bb1d988" \
+| xxd -r -p > policy.bin
+tpm2_createprimary -A o -g 0xb -C prim.ctx -G 0x1
+tpm2_create -c prim.ctx -g 0xb -G 0x1 -L policy.bin -o key.pub -O key.priv -E
+cmp -i 4:0 -n 32 key.pub policy.bin -s
```

Comment:
Same comment here as the others, should failures be reported this late in the game.